bookdown/02-basics-tasks.Rmd

```diff
@@ -80,6 +80,7 @@ mlr_tasks
 We can get a more informative summary of the example tasks by converting the dictionary to a `r ref("data.table()")` object:
 
 ```{r 02-basics-tasks-005}
+library(data.table)
```

Comment:
why is this added here?

Reply:
The book does not render without this on my machine. But if it just me, we can remove it.